### PR TITLE
WIP Stub unsupported Bean classes in JDK8 for improved javadoc

### PR DESCRIPTION
**Pull request by pshipton**
In com.sun.management stub:
DiagnosticCommandMBean
HotSpotDiagnosticMXBean
ThreadMXBean
VMOption

In sun.management stub:
DiagnosticCommandImpl

See also #7467

Depends on https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/340

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins compile plinux jdk8

**Comment by pshipton** [comment]
Will update the fix the copyright once the PR testing is complete.

File: jcl/src/jdk.management/share/classes/com/sun/management/HotSpotDiagnosticMXBean.java [hunk 1]
```diff
@@ -32,3 +33,4 @@
 public interface HotSpotDiagnosticMXBean extends PlatformManagedObject {
 
 }
+/*[ENDIF]*/
```
Comment (DanHeidinga) [inline jcl/src/jdk.management/share/classes/com/sun/management/HotSpotDiagnosticMXBean.java]: Can you also add a newline to end of this file?

**Comment by pshipton** [comment]
jenkins compile plinux jdk8

File: jcl/src/java.management/share/classes/sun/management/DiagnosticCommandImpl.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/*[INCLUDE-IF Sidecar18-SE-OpenJ9&!Sidecar19-SE]*/
```
Comment (DanHeidinga) [inline jcl/src/java.management/share/classes/sun/management/DiagnosticCommandImpl.java]: Why is is this stubbed out in 8 only?

Comment (pshipton) [reply jcl/src/java.management/share/classes/sun/management/DiagnosticCommandImpl.java]: The class was moved to com.sun.management.internal in Java 11+ and for Java 11+ it's already stubbed. 
https://github.com/eclipse/openj9/blob/master/jcl/src/jdk.management/share/classes/com/sun/management/internal/DiagnosticCommandImpl.java

**Comment by pshipton** [comment]
Need to stub sun/management/DiagnosticCommandImpl.c as well.

**Comment by pshipton** [comment]
jenkins compile plinux jdk8 depends ibmruntimes/openj9-openjdk-jdk8#340

**Comment by pshipton** [comment]
jenkins compile plinux jdk8 depends ibmruntimes/openj9-openjdk-jdk8#340

**Comment by pshipton** [comment]
I'm told these stubs don't have any affect on the javdoc so work on this is suspended.

**Pull request closed without merging**